terrawrap/utils/cli.py

```diff
@@ -34,7 +34,6 @@
     'try again later',
     'handshake timeout',
     'SSL_ERROR_SYSCALL',
-    'ConditionalCheckFailedException',
```

Comment:
Yes, this error can occur in other places, but it's also part of the error message for when the state is locked. Because of how the retrying logic is done right now, the network error logic takes precedence over the state lock logic, so Terrawrap ends up retrying the command over and over instead of unlocking the state and then retrying the command.

Reply:
We should have a story for redoing the retry logic in general, but for now this will fix more problems than it causes.